Common/ConvertUproot.py

```diff
@@ -1,44 +1,66 @@
 import uproot
 import awkward as ak
+import ROOT
 import os
 import numpy as np
-def saveFile(outFile, out_tree, treeName):
+
+
+def saveFile(outFile, out_tree_dict, histograms):
   if os.path.exists(outFile):
     os.remove(outFile)
-    with uproot.recreate(outFile, compression=uproot.LZMA(9)) as out_file:
-      out_file[treeName] = out_tree
-      out_file.close()
-    return outFile
+  with uproot.recreate(outFile, compression=uproot.LZMA(9)) as out_file:
+    for hist in histograms.keys():
+      out_file[hist] = histograms[hist]
+    for out_tree_key in out_tree_dict.keys():
+      out_file[out_tree_key] = out_tree_dict[out_tree_key]
+  out_file.close()
```

Comment:
You are using `with` above. `close` should be redundant.

Reply:
ok